grails-app/services/com/recomdata/transmart/data/export/HighDimExportService.groovy

```diff
@@ -6,6 +6,8 @@ import org.transmartproject.core.dataquery.highdim.AssayColumn
 import org.transmartproject.core.dataquery.highdim.HighDimensionDataTypeResource
 import org.transmartproject.core.dataquery.highdim.assayconstraints.AssayConstraint
 import org.transmartproject.core.dataquery.highdim.projections.Projection
+import org.transmartproject.core.ontology.OntologyTerm
+import org.transmartproject.db.concept.ConceptFullName
```

Comment:
Don't depend on compile-time on core-db.

Perhaps it would be time to move ConceptFullName and ConceptKey to core-api. I've felt the need to use them outside core-db a lot of times.